api.raml

```diff
@@ -384,8 +407,20 @@ schemas:
         responses:
           200:
             description: |
-              The password for the user with that `employee_id` was
-              successfully updated.
+              The password for the user with that `id` was successfully updated.
+          404:
```

Comment:
This might be a `204` to match our use of 404 vs. 204 elsewhere in the spec, but thanks for highlighting this. I hadn't thought about what happens when we try to update the password of a user that isn't found in the database.